common/game/GamePlayer.h

```diff
@@ -23,7 +23,7 @@
 
 
 class GamePlayer {
-protected:
```

Comment:
im fine with this but i didn't see any direct field accesses so idk if you actually need this